tree/dataframe/inc/ROOT/RDF/RCutFlowReport.hxx

```diff
@@ -41,7 +41,7 @@ public:
    const std::string &GetName() const { return fName; }
    ULong64_t GetAll() const { return fAll; }
    ULong64_t GetPass() const { return fPass; }
-   float GetEff() const { return 100.f * (fPass / float(fAll)); }
+   float GetEff() const { return 100.f * (float(fPass) / float(fAll)); }
```

Comment:
i'm not sure this does anything, godbolt seems to [confirm](https://godbolt.org/z/dDaq-5)

Reply:
You are right :) !